modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/HTMLReport.pm

```diff
@@ -95,15 +100,19 @@ sub fetch_input {
             $sums[0] += $species->get_value_for_tag('nb_genes');
             $sums[1] += $species->get_value_for_tag('nb_seq');
             $sums[2] += $species->get_value_for_tag('nb_orphan_genes');
-            $sums[3] += $species->get_value_for_tag('nb_genes_in_tree');
-            $sums[4] += $species->get_value_for_tag('nb_genes_in_tree_single_species');
-            $sums[5] += $species->get_value_for_tag('nb_genes_in_tree_multi_species');
+            $sums[3] += $species->get_value_for_tag('nb_genes_in_unfiltered_cluster');
+            $sums[4] += $species->get_value_for_tag('nb_genes_unassigned');
+            $sums[5] += $species->get_value_for_tag('nb_genes_in_tree');
+            $sums[6] += $species->get_value_for_tag('nb_genes_in_tree_single_species');
+            $sums[7] += $species->get_value_for_tag('nb_genes_in_tree_multi_species');
```

Comment:
I think [this](https://github.com/Ensembl/ensembl-webcode/blob/release/104/modules/EnsEMBL/Web/Document/HTML/Compara/GeneTrees.pm) `ensemb-webcode` module will have to be updated as well.

Reply:
Have looked at this following your comment @ipilizota , and it will not break the webcode, so it would still be safe to integrate our side. But indeed web team will want to update [these subs](https://github.com/Ensembl/ensembl-webcode/blob/release/104/modules/EnsEMBL/Web/Document/HTML/Compara/GeneTrees.pm#L264-L367) to reflect the new tag `nb_genes_unassigned` instead of `nb_orphan_genes` with change of column name included. -Although, not entirely sure the webteam usage is incorrect either because they are technically still orphaned genes being reported, and as I can see, they are not being used in any calculations as we were doing on our side :shrug: